inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -281,23 +291,19 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 						// @ts-expect-error
 						window.repo = newRepo
 					}
-
-					if (newRepo.errors().length > 0) {
-						setLixErrors(newRepo.errors())
-						return
-					} else {
-						setLixErrors([])
-					}
+					// TODO replace this with `setLixErrors([])` as soon as repo throws
```

Comment:
@martin-lysk  what is the impact of this? is this a precondition to mergin?

Reply:
maybe its just a forgotten comment

Reply:
Good catch, That was meant to fix the catching in openRepo - which it didn't because of its async behaviour :-( now since openRepo throws - this is was removed with the recent push